README.md

```diff
@@ -43,6 +43,12 @@ To add custom origin header in the request use `-o` option
 $ python3 cswsh.py "wss://echo.websocket.org" -o "http://localhost:8080"
 ```
 
+#### Disable SSL certificate verification
+If you don't want the tool to verify the server certificate, use `-i`/`--no-cert-check` option
```

Comment:
I'm using the shorthand notation in the entire README so do the suggested change for consistency.
```suggestion
If you don't want the tool to verify the server certificate, use `-i` option
```

Reply:
I see. I've updated the README.